docs/content/configuration/indexing-service.md

```diff
@@ -251,6 +251,7 @@ Middle managers pass their configurations down to their child peons. The middle
 |`druid.indexer.runner.javaOpts`|-X Java options to run the peon in its own JVM.|""|
 |`druid.indexer.runner.maxZnodeBytes`|The maximum size Znode in bytes that can be created in Zookeeper.|524288|
 |`druid.indexer.runner.startPort`|The port that peons begin running on.|8100|
+|`druid.indexer.runner.separateIngestionServerRequired`|Use separate server and consequently separate jetty thread pool for ingesting events|false|
```

Comment:
`separateIngestionEndpoint` might be clearer. "server" makes it seem it's a different component entirely, and "required" does not add much information.